README.md

```diff
@@ -592,18 +592,20 @@ Trees have the ability to nest.
 
 ```go
 t := tree.Root(".").
-  Child("Item 1").
-  Child(
-    tree.Root("Item 2").
-      Child("Item 2.1").
-      Child("Item 2.2").
-      Child("Item 2.3"),
-  ).
-  Child(
-    tree.Root("Item 3").
-      Child("Item 3.1").
-      Child("Item 3.2"),
-  )
+    Child("macOS").
```

Comment:
Heads up the alignment in the lists section is two spaces instead of tabs, so we should update the indentation to match

Reply:
Good catch. The rest of Lip Gloss uses four spaces for tabs so I normalized the list section to four tabs instead.